class_1/practice_activites.py

```diff
@@ -85,24 +155,46 @@
 print x + " " + y
 print 2 * x
 print x + str(5) + y
-print x + 5 + y
 print len(x) + len(y)
 print x == "hello"
-print y == "WORLD"'''
+print y == "WORLD"
 
-# Using variables x and y write a statement that will print out "Coding is easy"
-'''x = "oding"
-y = "is"'''
+# TODO: Using variables x and y write a statement that will print out "Coding is easy"
 
-# Activity 8
-# Using variable x and only addition and subtraction make the following statements true
-'''x = "Fun"
+print "- What do you think about coding?"
+x = "oding"
+y = "is"
+
+
+
+# Activity 7
+print("--------------------------------------------------")
+print("Activity 7")
+print("--------------------------------------------------")
+
+# TODO: Using variable x and only addition and subtraction make the following statements true
```

Comment:
i would make each of these TODO's say : 'make each of the following statements true'.    When i read it now i think that  a single value should make both true.